Cargo.toml

```diff
@@ -19,6 +19,21 @@ targets = [
   "x86_64-apple-darwin",
 ]
 
+[package.metadata.packager]
+name = "verso"
+product-name = "verso"
+identifier = "com.crabnebula.verso"
```

Comment:
Could we skip this identifier? I don't think it's okay to make this belong to CN.

Reply:
We need some identifier. You can suggest your own.

Reply:
`eu.browser.verso` then

Reply:
I hope `com.eu-browser.verso` is fine?